components/Elevation/src/UIColor+MaterialElevation.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UIColor (MaterialElevation)
+
+/// Returns a color that takes the specified elevation value into account.
```

Comment:
Nit: Please use block-style comments (`/** */`) for multi-line comments in Objective-C to match the convention in our library.

Reply:
Addressed in ec78c1cf6e8932e4ac318e277dae7e31646bcab5